com/duke/storage/Storage.java

```diff
@@ -0,0 +1,64 @@
+package com.duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
+    private static String FILE_PATH = "src/main/data/input.txt";
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public void printFileContents() {
```

Comment:
Should there be a javadoc description for this public method? I noticed this in the other methods also.